Run-time panic if file or dir doesn't exist

```sh
iliaf@ilia-arco-vm ~/g/s/g/i/go-du (main)> ./dist/go-du_linux_amd64/go-du ,.
stat ,.: no such file or directory
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x4b814d]

goroutine 1 [running]:
github.com/iliafrenkel/go-du/app/dirtree.(*DirTree).buildDirTree(0xc0000541e0)
	/home/iliaf/go/src/github.com/iliafrenkel/go-du/app/dirtree/dirtree.go:67 +0xcd
github.com/iliafrenkel/go-du/app/dirtree.New(0x7ffe1da3534d, 0x2, 0x200, 0x60)
	/home/iliaf/go/src/github.com/iliafrenkel/go-du/app/dirtree/dirtree.go:52 +0x8f
main.main()
	/home/iliaf/go/src/github.com/iliafrenkel/go-du/app/main.go:144 +0x11d
```

--- a/.gitignore
+++ b/.gitignore
@@ -13,6 +13,8 @@ testdata
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+cover.html
+coverage.txt
 
 # Dependency directories (remove the comment below to include it)
 # vendor/

--- a/app/dirtree/dirtree.go
+++ b/app/dirtree/dirtree.go
@@ -63,6 +63,7 @@ func (dt *DirTree) buildDirTree() {
 	dtInfo, err := os.Stat(dt.path)
 	if err != nil {
 		errLog.Println(err)
+		return
 	}
 	dt.size = dt.calcSize(dtInfo.Size())
 	if !dtInfo.IsDir() {
@@ -126,6 +127,9 @@ func (dt *DirTree) PrintDirTree(outFormat string, countFiles bool, summarise boo
 // not the file size we need calculate the number of filesystem blocks
 // allocated to the file.
 func (dt *DirTree) calcSize(size int64) int64 {
+	if size == 0 {
+		return 0
+	}
 	allocSize := (1 + (size-1)/dt.blockSize) * dt.blockSize
 	return 1 + (allocSize-1)/dt.unitSize
 }

--- a/app/dirtree/dirtree_test.go
+++ b/app/dirtree/dirtree_test.go
@@ -48,6 +48,23 @@ type testCase struct {
 
 var testCases = []testCase{
 	{
+		name:  "File does not exist",
+		files: []testFile{},
+		path:  "./ak5i8fg74",
+		opts: options{
+			BlockSize:       false,
+			CountFiles:      false,
+			DereferenceAll:  false,
+			DereferenceArgs: false,
+			OneFileSystem:   false,
+			Summarise:       false,
+		},
+		expected: 0,
+		output: []string{
+			"0\t./ak5i8fg74",
+		},
+	},
+	{
 		name:  "Access denied",
 		files: []testFile{},
 		path:  "/root",
@@ -59,7 +76,7 @@ var testCases = []testCase{
 			OneFileSystem:   false,
 			Summarise:       false,
 		},
-		expected: 0,
+		expected: 8,
 		output: []string{
 			"8\t/root",
 		},

--- a/app/main_test.go
+++ b/app/main_test.go
@@ -52,3 +52,8 @@ func Test_FlagConflicts(t *testing.T) {
 		t.Errorf("Expecting conflict between -a and -s flags.")
 	}
 }
+
+func Test_PrintVersion(t *testing.T) {
+	// I don't know what to test here yet.
+	printVersion()
+}

